Content.Client/UserInterface/Systems/Viewport/ViewportUIController.cs

```diff
@@ -15,10 +16,11 @@ public sealed class ViewportUIController : UIController
     [Dependency] private readonly IPlayerManager _playerMan = default!;
     [Dependency] private readonly IEntityManager _entMan = default!;
     [Dependency] private readonly IConfigurationManager _configurationManager = default!;
+    [Dependency] private readonly IConsoleHost _conHost = default!;
 
-    public static readonly Vector2i ViewportSize = (EyeManager.PixelsPerMeter * 21, EyeManager.PixelsPerMeter * 15);
     public const int ViewportHeight = 15;
-    private MainViewport? Viewport => UIManager.ActiveScreen?.GetWidget<MainViewport>();
+
+    public MainViewport? Viewport => (UIManager.ActiveScreen?.GetWidget<MainViewport>() ?? UIManager.ActiveScreen?.GetWidget<SplitViewportWidget>());
```

Comment:
AFAIK widgets don't support interfaces or inheritance, so you just have to do stuff like this.